File: modules/x-engine-native-hummer/iOS/Class/Native_hummer.m
//
//  Native_hummer.m
//  hummer
//


#import "Native_hummer.h"
#import "XENativeContext.h"
#import <Hummer/Hummer.h>

@interface Native_hummer()
{ }
@end

@implementation Native_hummer
NATIVE_MODULE(Native_hummer)

 - (NSString*) moduleId{
    return @"com.zkty.native.hummer";
}

- (int) order{
    return 0;
}

- (void)afterAllNativeModuleInited{
    [Hummer startEngine:nil];
 
} 
-(NSString*) test{
    return @"test";
}
@end
 

File: modules/x-engine-native-hummer/android/native-hummer/src/main/java/com/zkty/nativ/hummer/Nativehummer.java
package com.zkty.nativ.hummer;

import android.util.Log;

import com.tencent.mmkv.MMKV;
import com.zkty.nativ.core.NativeModule;
import com.zkty.nativ.core.XEngineApplication;

import java.io.File;

public class Nativehummer extends NativeModule implements Ihummer {

    @Override
    public String moduleId() {
        return "com.zkty.native.hummer";
    }

    @Override
    public void afterAllNativeModuleInited() {

    }
}
